desktop/components/email/client/editorial_signup.coffee

```diff
@@ -20,21 +21,43 @@ module.exports = class EditorialSignupView extends Backbone.View
     'click .modal-bg': 'hideEditorialCTA'
     'click .cta-bar-defer': 'hideEditorialCTA'
 
-  initialize: ({isArticle = false}) ->
+  initialize: ({isArticle = false, @isABTest = false}) ->
+    @params = qs.parse(location.search.replace(/^\?/, ''))
     @setupAEArticlePage() if isArticle
     @setupAEMagazinePage() if @inAEMagazinePage()
+    @revealArticlePopup = _.once(@revealArticlePopup)
+    mediator.on 'modal:closed', @setDismissCookie
+    mediator.on 'auth:sign_up:success', @setDismissCookie
```

Comment:
I'd propose a name change here to something that makes it more clear that this is used to identify that we don't want to show the cta again. I don't think it should block the PR though.